src/shared/ui/CloseMenuIcon/index.tsx

```diff
@@ -0,0 +1,2 @@
+import CloseMenuIcon from "./CloseMenuIcon";
+export default CloseMenuIcon;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.